src/sinks/prometheus/remote_write.rs

```diff
@@ -41,6 +45,9 @@ pub(crate) struct RemoteWriteConfig {
     #[serde(default)]
     pub request: TowerRequestConfig,
 
+    #[serde(default)]
+    pub tenant_id: Option<String>,
```

Comment:
Is there a reason why this can't be:

```suggestion
    pub tenant_id: Option<Template>,
```
similar to the `tenant_id` field in the loki sink?

Reply:
Um, no, I'm not sure why I left it a `String`. Will fix.